dev_tools/scripts/gdb-ginkgo.py

```diff
@@ -106,14 +106,11 @@ def __init__(self, val):
         self.val = val
         self.execname = str(self.val['exec_']['_M_ptr'].dereference().dynamic_type)
         self.pointer = get_unique_ptr_data_ptr(self.val['data_']);
-        self.is_cpu = re.match('gko::(Reference|Omp)Executor', str(self.execname)) is not None
 
     def children(self):
-        if self.is_cpu:
-            return self._iterator(self.pointer, self.val['num_elems_'])
-        return []
+        return self._iterator(self.pointer, self.val['num_elems_'])
```

Comment:
using gdb only in cuda not hip?

Reply:
Yes, that probably makes sense. This is still left-over from when I thought AMD would work